README.md

```diff
@@ -13,7 +13,7 @@ manifests, image indexes (multi-arch images) and custom artifacts
 
 In addition to Bazel rules, we offer many helpers for interacting with OCI
 artifacts under the `go/pkg` directory and a CLI tool for creating new OCI
-artifacts. You can add the Go library as a dependency by running `go get github.com/DataDog/rules_oci/go@latest`.
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** Thanks for making this change!